### PR TITLE
Allow skipping RBAC spec when instlling/upgrading Admin Console

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: cmd/kots/cli/admin-console-upgrade.go [hunk 1]
```diff
@@ -96,9 +97,11 @@ func AdminConsoleUpgradeCmd() *cobra.Command {
 	cmd.Flags().String("registry-password", "", "password to use to authenticate with the registry")
 	cmd.Flags().String("kotsadm-namespace", "", "set to override the namespace of kotsadm image. this may create an incompatible deployment because the version of kots and kotsadm are designed to work together")
 	cmd.Flags().String("wait-duration", "2m", "timeout out to be used while waiting for individual components to be ready.  must be in Go duration format (eg: 10s, 2m)")
+	cmd.Flags().Bool("ensure-rbac", true, "when set, kots will create the roles and rolebindings necessary to manage applications")
 	cmd.Flags().MarkHidden("force-upgrade-kurl")
 	cmd.Flags().MarkHidden("kotsadm-tag")
 	cmd.Flags().MarkHidden("kotsadm-namespace")
+	cmd.Flags().MarkHidden("ensure-rbac")
```
Comment (emosbaugh) [inline cmd/kots/cli/admin-console-upgrade.go:104]: does this need to be hidden?

File: cmd/kots/cli/install.go [hunk 2]
```diff
@@ -399,6 +400,9 @@ func InstallCmd() *cobra.Command {
 	cmd.Flags().MarkHidden("with-dockerdistribution")
 	cmd.Flags().MarkHidden("storage-base-uri-plainhttp")
 
+	cmd.Flags().Bool("ensure-rbac", true, "when set, kots will create the roles and rolebindings necessary to manage applications")
+	cmd.Flags().MarkHidden("ensure-rbac")
```
Comment (emosbaugh) [inline cmd/kots/cli/install.go:404]: does this need to be hidden?

**Review by emosbaugh — APPROVED** [review]

File: cmd/kots/cli/install.go (hunk 2 above)
Comment (divolgin) [reply cmd/kots/cli/install.go:404]: I think we should keep this hidden and possibly remove it once we have a proper fix

Comment (emosbaugh) [reply cmd/kots/cli/install.go:404]: tbh i think this is a pretty good solution. we can do a better job with minimal rbac support but this seems different.

**Pull request merged**